react-native/make-shim.js

```diff
@@ -18,8 +18,10 @@ try {
   fs.writeFileSync(path.join(root, 'package.json'), `{
   "main": "index.js"
 }
-`) } catch (_) {}
+`)
+} catch (_) {}
```

Comment:
We don't check for existence and things like that which would be caught. I think in practice this thing never fails and if it did you're node_modules wouldn't work